docs/adr/0016-reports.md

```diff
@@ -0,0 +1,21 @@
+# 16. Reports
+Date: 2023-06-16
+
+## Status
+Accepted
+
+## Context
+An OSIv2 analysis produces reports, so as part of moving OSI into job-server we need to also support reports here.
+
+
+## Decision
+We will support reports in job-server by mirroring the implementation of those in the Reports service, with the exception that the underlying ReleaseFile is related to a Report via a ForeignKey.
+
+We are defining a Report on the OpenSAFELY platform as:
+
+    A released file wrapped with some extra metadata and styling.
+
+
```

Comment:
We also made some decisions about the styling that it might be worth adding a sentence about here. IIRC, we expect the report to be decorated with the standard CSS classes, with Job Server providing the stylesheet and therefore providing a mechanism for some consistent styling across the reports (although of course, this can be overridden with inline styles if needed, I believe?).

Reply:
Updated to add this without being prescriptive on the classes so we don't feel we're tied to the current set.

Reply:
I think that's sensible, it was the division of responsibility that I thought was important to mention, which I can see you've now done. Thank you :)